pandas/core/series.py

```diff
@@ -195,9 +196,12 @@ def __init__(self, data=None, index=None, dtype=None, name=None,
                 else:
                     data = data.reindex(index, copy=copy)
             elif isinstance(data, Categorical):
-                if dtype is not None:
+                # Allow dtype=category only, otherwise error
```

Comment:
Fixed others, this is the last one. Do you mean ensuring this is handled? 
```
a = pd.Series([1,2,3], dtype='category')
```
Because that seems graceful:
```
>>> a
0    1
1    2
2    3
dtype: category
Categories (3, int64): [1, 2, 3]
```